scripts/mail-alarm

```diff
@@ -99,17 +100,44 @@ def get_sender(other_config):
     if other_config.has_key('mail-sender'):
         return other_config['mail-sender']
 
+def get_mail_language(other_config):
+    if other_config.has_key('mail-language'):
+        return other_config['mail-language']
+    else:
+        return "en-US"
+
 def get_config_file():
     try:
         return open('/etc/mail-alarm.conf').read()
     except:
         return default_config
 
-class EmailTextGenerator:
-    pass
+class EmailTextGenerator(object):
+    def __init__(self, mail_language, mail_language_component):
+        self.mail_language = mail_language
+        self.mail_language_component = mail_language_component
+
+    def mail_err(self, key):
+        log_err("[%s] error in looking for (%s:%s)." % (self.__class__.__name__, self.mail_language_component, key))
+        log_err("Mail language pack: %s." % (self.mail_language))
+
+    def mail_subject(self):
+        try:
+            return self.mail_language['mail-alarms'][self.mail_language_component]['subject']
+        except:
```

Comment:
I would prefer to catch the expected error here and fail if something unexpected is happening:
```python
    except KeyError:
        mail_err('subject')
        return None
    except Exception as err:
        log_err("[%s] unexpected failure: %s" % (self.__class__.__name__, str(err))
        raise  # return None if you prefer not to fail completely
```